app/src/locales/en.json

```diff
@@ -44,7 +44,8 @@
   "defaultProfile": "Default profile",
   "importNewProfile": "Import a new profile",
   "selectFhirProfilePrompt": "Please select a profile for the",
-  "resource": "resource",
+  "resource": "{{count}} resource",
```

Comment:
I just saw that this key is used elsewhere, but with no count... 
Maybe you'll have to use another key than `resource`

Reply:
The `resource` key was already used in `FhirProfileStep` component